docs/build.sbt

```diff
@@ -1 +1 @@
-crossScalaVersions ~= { oldVersions => oldVersions.filter(_.startsWith("2.12")) }
+crossScalaVersions ~= { oldVersions => oldVersions.filterNot(_.startsWith("2.11")) }
```

Comment:
Not related to this change in particular, but do we need to have `crossScalaVersions` defined for the `docs` project? I assume we're always going to build the site using a specific Scala version.

Reply:
We don't really need it, but even though we always publish the site using a specific version cross compiling ensures that our documentation works with both Scala 2.12 and 2.13.

Reply:
We should then start running `++docs/mdoc` in the CI, otherwise we will only build the documentation for Scala 2.12.

Reply:
You're right, we aren't cross-compiling in CI. I think I didn't do it at the time because the deprecation warnings would make the build fail, but I manually did `++docs/mdoc` when I was migrating to make sure that was the case.